stacks/api/activity/services/src/activity.interceptor.ts

```diff
@@ -43,7 +43,7 @@ export function getUser(user: any): string {
   return user ? user._id : undefined;
 }
 
-export function createActivity(
+export async function createActivity(
```

Comment:
Returning a promise is better than make this function void I think. Why did we need to make this change?

Reply:
I have made it this way because I don't want this function to return the result of the insertMany.

Reply:
since it is an async function it returns `Promise<void>` since I didn't return anything.